src/main/java/com/deavensoft/paketomat/center/CenterServiceImpl.java

```diff
@@ -1,23 +1,31 @@
 package com.deavensoft.paketomat.center;
 
 import com.deavensoft.paketomat.center.model.Package;
+import com.deavensoft.paketomat.center.model.Paid;
 import com.deavensoft.paketomat.center.model.Status;
+import com.deavensoft.paketomat.email.EmailDetails;
+import com.deavensoft.paketomat.email.EmailService;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 import java.time.LocalDateTime;
+
+import java.security.SecureRandom;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 
 @Slf4j
 @Service
 public class CenterServiceImpl implements CenterService {
     private final CenterRepository centerRepository;
-
+    private final EmailService emailService;
     @Autowired
-    public CenterServiceImpl(@Qualifier("center") CenterRepository centerRepository) {
+    public CenterServiceImpl(@Qualifier("center") CenterRepository centerRepository,EmailService emailService) {
         this.centerRepository = centerRepository;
+        this.emailService=emailService;
```

Comment:
Delete the constructor and use Lombok's annotation.

Reply:
Done.

Reply:
Annotation?